packages/flutter/lib/src/material/selectable_text.dart

```diff
@@ -359,9 +358,16 @@ class SelectableText extends StatefulWidget {
   /// {@macro flutter.widgets.editableText.cursorRadius}
   final Radius? cursorRadius;
 
-  /// The color to use when painting the cursor.
+  /// The color of the cursor.
   ///
-  /// Defaults to the theme's `cursorColor` when null.
+  /// The cursor indicates the current location of the text insertion point in
```

Comment:
The cursor indicates the current location of the text insertion point in the field. => 
The cursor indicates the current text insertion point.